browser-sync/browser-sync.d.ts

```diff
@@ -334,6 +334,8 @@ declare module "browser-sync" {
              * @param name the identifier used for retrieval
              */
             get(name: string): BrowserSyncInstance;
+
+            instances: BrowserSyncInstance[];
```

Comment:
It's not in the documentation, but it is exported [here](https://github.com/BrowserSync/browser-sync/blob/master/index.js#L358).

I have asked about it [here](https://github.com/BrowserSync/browser-sync/issues/839#issuecomment-161241122).